aim/cli/watcher_cli.py

```diff
@@ -0,0 +1,171 @@
+import click
+from click import core
+import uuid
+import json
+
+from aim.ext.notifier import get_config
+from aim.ext.notifier.utils import has_watcher_config, set_default_config
+from aim.sdk.run_status_watcher import RunStatusWatcher
+from aim.sdk.repo import Repo
+
+core._verify_python3_env = lambda: None
+DEFAULT_MESSAGE_TEMPLATE = "❗️ Something wrong with Run '{run_hash}'. Please check. ❗️"
+
+
+def check_configuration(repo: Repo):
+    if not has_watcher_config(repo.path):
+        if click.confirm(f'Repo {repo.path} has no configured notifications. Would you like to '
+                         f'use default configuration?', abort=True):
+            set_default_config(repo.path)
+
+
+@click.group(invoke_without_command=True)
+@click.option('--repo', required=False, type=click.Path(exists=True,
+                                                        file_okay=False,
+                                                        dir_okay=True,
+                                                        writable=True))
+@click.pass_context
+def cli_entry_point(ctx, repo):
+    repo_path = repo or Repo.default_repo_path()
+    if ctx.invoked_subcommand is None:
```

Comment:
what about considering moving `watcher.start_watcher()` under a separate `up` command?
I usually run a tool's cli entrypoint to see help message and available commands. when I run `aim-watcher` the server started working and the only thing I saw was an empty row, which was quite confusing.